code/game/objects/items/pondering_orb.dm

```diff
@@ -0,0 +1,26 @@
+/obj/item/pondering_orb
+	name = "Pondering Orb"
+	icon = 'icons/obj/pondering_orb.dmi'
+	icon_state = "orb_of_ponders"
+	desc = "Born to ponder."
+
+/obj/item/pondering_orb/attack_hand(mob/user, list/modifiers)
+	balloon_alert(user, "You start pondering the orb...")
+	balloon_alert_to_viewers("[user.name] starts to ponder the orb...")
```

Comment:
```suggestion
	balloon_alert_to_viewers("pondering the orb...")
```

1. This would've thrown both balloon alerts
2. Balloon alerts should be kept succinct, and generally avoid names etc.